CMakeLists.txt

```diff
@@ -229,7 +229,8 @@ add_custom_target(pip_install_requirements
 # the package does not change, but the code did change. We need --no-deps,
 # because otherwise pip would also force reinstall all dependencies.
 add_custom_target(pip_install
-    python3 -m pip install --force-reinstall --no-deps ${PYINSTALLOPTIONS} "${pyAMReX_BINARY_DIR}/amrex*whl"
+    ${CMAKE_COMMAND} -E env AMREX_MPI=${AMReX_MPI}
+        python3 -m pip install --force-reinstall --no-deps ${PYINSTALLOPTIONS} "${pyAMReX_BINARY_DIR}/amrex*whl"
```

Comment:
The problem is probably that this is called in a `cmd` shell for this test :thinking:

Reply:
Oh, it does not... https://superuser.com/a/460609/701133